### DISCUSSION TITLE
Possible GPL Violation by CxStudio app.

### DISCUSSION THREAD

**Discussion by Chuvi-w** [termux/termux-app#3428] [General]
I've found [this app](https://play.google.com/store/apps/details?id=com.alif.ide.cpp).
Some files of it contains string `/home/builder/.termux-build`, which probably indicates that this project used the Termux source code.

Source code of CxStudio is not awayable

**Comment by agnostic-apollo** [comment]
The `termux-packages` build infrastructure is Apache licensed, so they don't need to reveal sources.

https://github.com/termux/termux-packages/blob/master/LICENSE.md

If its using termux `app` or `termux-shared` components under GPLv3, only then it would be a violation. The terminal itself is apache licensed.
